### DISCUSSION TITLE
Updating items properly after edit in item_template

### DISCUSSION THREAD

**Discussion by kapstok** [azerothcore/azerothcore-wotlk#12617] [Guides & Tips]
I have two already existing items in WoW (item A and item B).
Item A is held in one of my character's inventory while the other item is only a random equipable item I found in my item_template table.
I edit both rows of items A and B in the item_template table (e.g. change their name and dps). After that, I destroy item A from my inventory.

If I get both items in WoW again (e.g. by looting of buying them), item A isn't updated while item B is. My suggestion is to add a small bit of information to the Wiki which explains how to show item A again as prescribed by the item_template table.

It is a question I'm wondering myself, but I presume that others might deal (or have dealt) with the similar problem.

**Comment by Nyeriah** [comment]
You must clear your client cache after making item_template changes for the new values to be displayed, so you must close your game completely, navigate to the cache folder and delete it. 

If the item wasn't cached then it will display fine (rarely the case)

**Reply by kapstok** [reply]
Thanks for the quick response. I'll try that out :)
Still something to add to wiki? If not, the discussion can be closed
